src/java.base/share/classes/java/util/concurrent/DelayQueue.java

```diff
@@ -51,8 +51,7 @@
  * {@code Delayed} elements, in which an element can only be taken
  * when its delay has expired.  The <em>head</em> of the queue is that
  * {@code Delayed} element whose delay expired furthest in the
- * past.  If no delay has expired there is no head and {@code poll}
```

Comment:
I like introducing the concept of "expired head".
Aren't we defining "expired head" here, not "head"?

Reply:
@Martin-Buchholz I read the line before the removed line (The <em>head</em> of the queue is that {@code Delayed} element whose delay expired furthest in the past.) as defining that the head will be the thing with the "soonest" expiry, which is important if one is used to LIFO or FIFO. Any idea on how we could improve that sentence? 🤔